app/browser/menu.js

```diff
@@ -363,13 +365,17 @@ const createHistorySubmenu = (CommonMenu) => {
   return submenu
 }
 
-const createBookmarksSubmenu = (CommonMenu) => {
+const isCurrentLocationBookmarked = () => {
```

Comment:
Super nice cleanup :smile: The previous code was hacky (because it could be changed in appStore OR in the navigationBar control) and there were definitely ways to cause it to fail (ex: shows checked but its not bookmarked)